core/src/main/java/com/novoda/noplayer/Player.java

```diff
@@ -33,8 +33,18 @@
 
     void selectAudioTrack(PlayerAudioTrack audioTrack);
 
+    boolean hasAvailableSubtitles();
+
+    void selectSubtitleTrack(PlayerSubtitleTrack subtitleTrack, PlayerSubtitlesView playerSubtitlesView);
+
+    void selectFirstAvailableSubtitleTrack(PlayerSubtitlesView playerSubtitlesView);
```

Comment:
I agree. We should push for symmetry, I'd rather avoid having lots of helper methods that is going to make the Player interface bloat.